app/views/practices/implementation_forms/_optional_attachment_link_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<% resource ||= nil
+   placeholder ||= 'RANDOM_NUMBER_OR_SOMETHING'
+   placeholder = placeholder.to_s + '_link'
+   area = area.to_s
+%>
+
+<div id="<%= area %>_link_form<%= "_#{placeholder}" if resource %>" <% if resource %> class="resource_container margin-bottom-5" <% end %>>
+  <input type="hidden" name="practice[practice_resources_attributes][<%= placeholder %>][media_type]" value="link"/>
+  <input type="hidden" name="practice[practice_resources_attributes][<%= placeholder %>][resource_type]" value="optional"/>
+  <% if resource&.id  %>
+    <input value="<%= resource&.id %>" type="hidden" name="practice[practice_resources_attributes][<%= placeholder %>][id]"/>
+  <% end %>
+  <div class="<%= area %>-input-container">
+    <%= label_tag "practice_#{area}_link", 'Link (paste the full address)', class: 'usa-label overview-resource-label', for: "practice_#{area}_attributes_#{placeholder}_link_link_url" %>
+    <%= render partial: 'practices/overview_error_partials/link', locals: {
+        area: area,
+        resource_type: 'link',
+        message: 'Please enter a valid url/link'
+    }
+    %>
+
+    <input <%= resource&.id ? 'required' : '' %> class="usa-input" value="<%= resource&.link_url %>" type="text" name="practice[practice_resources_attributes][<%= placeholder %>][link_url]" id="practice_<%= area %>_attributes_<%= placeholder %>_link_url" style="width: 643px"/>
+  </div>
+  <div class="<%= area %>-input-container">
+    <%= label_tag "practice_#{area}_name", 'Title', class: 'usa-label overview-resource-label', for: "practice_#{area}_attributes_#{placeholder}_link_name" %>
+    <%= render partial: 'practices/overview_error_partials/name', locals: {
+        area: area,
+        resource_type: 'link',
+        message: 'Please enter a link title'
+    }
+    %>
+    <input <%= resource&.id ? 'required' : '' %> class="usa-input" value="<%= resource&.name %>" type="text" name="practice[practice_resources_attributes][<%= placeholder %>][name]" id="practice_<%= area %>_attributes_<%= placeholder %>_name" style="width: 393px"/>
+  </div>
+
+  <div class="<%= area %>-input-container">
+    <%= label_tag "practice_#{area}_description", 'Description', class: 'usa-label overview-resource-label', for: "practice_#{area}_attributes_#{placeholder}_link_description" %>
+    <%= render partial: 'practices/overview_error_partials/description', locals: {
+        area: area,
+        resource_type: 'link',
+        message: 'Please enter a link description'
+    }
+    %>
+    <input <%= resource&.id ? 'required' : '' %> class="usa-input" value="<%= resource&.description %>" type="text" name="practice[practice_resources_attributes][<%= placeholder %>][description]" id="practice_<%= area %>_attributes_<%= placeholder %>_description" style="width: 643px"/>
```

Comment:
i see a lot of these `style=width: ...px` attributes. these should be using the `grid-col-...` classes in uswds

Reply:
These have been updated to use uswds grid-cols